wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,261 @@
+[o!s]: /wiki/shared/mode/osu.png 
+[o!t]: /wiki/shared/mode/taiko.png 
+[o!c]: /wiki/shared/mode/catch.png 
+[o!m]: /wiki/shared/mode/mania.png 
+
+# Skins Compendium
+
```

Comment:
add a new line and write `Compendium of all completed osu! skins.` or something like this